siliconcompiler/libs/sky130io.py

```diff
@@ -34,6 +34,11 @@ def setup(chip):
     lib.add('output', stackup, 'gds', os.path.join(libdir, 'sky130_fd_io.gds'))
     lib.add('output', stackup, 'gds', os.path.join(libdir, 'sky130_ef_io__gpiov2_pad_wrapped.gds'))
 
+    lib.set('asic', 'cells', 'filler', ['sky130_ef_io__com_bus_slice_1um',
```

Comment:
Neat use of the fill cell parameter - I'm guessing these are the cells that fill the gaps between individual pads?

Reply:
yeah, this will only work on older nodes where the fillers are the same for the different pad orientations